lib/app/modules/smart/standalone_launch_sequence.rb

```diff
@@ -96,6 +104,9 @@ class StandaloneLaunchSequence < SequenceBase
           )
         end
 
+        # Confirm that there is a @params object from the redirect
+        assert !@params.nil?, OAUTH_REDIRECT_FAILED
```

Comment:
Should this assertion pass if `@params` is an empty object?

Reply:
I believe the future asserts will fail (with a more accurate error message), indicating desired fields don't exist in the hash